ci/baremetal/baremetal-cluster.lokocfg.envsubst

```diff
@@ -35,4 +35,9 @@ cluster "bare-metal" {
   # Acts as a smoke test to check if API server is functional after addition
   # of extra flags.
   oidc {}
+
+  labels = {
```

Comment:
Should we add a simple test that checks labels are there (without any reboots)?

Reply:
`TestSelfHostedKubeletLabels` will already test that, though we could perhaps have one more test to just seek for those labels on the nodes. I'm not sure if we should implement here though, what do you think?

Reply:
I was thinking that since that we start skipping that test here we should add a simple labels test to not lose coverage. Does that make sense?

Reply:
Ah yes, I checked the code on the wrong branch. let me implement it then.